include/swift/IDE/CodeCompletion.h

```diff
@@ -140,27 +140,27 @@ class CodeCompletionStringChunk {
 
     /// The first chunk of a substring that describes the parameter for a
     /// function call.
-    CallParameterBegin,
```

Comment:
We should update the doc comments for this and all the other cases too.

Reply:
Updated the comments